frontend/src/pages/org/settings/components/billing.ts

```diff
@@ -124,18 +128,35 @@ export class OrgSettingsBilling extends BtrixElement {
                               class="text-base"
                             ></sl-icon>
                             <span>
-                              ${msg(
-                                html`Your plan will be canceled on
-                                  <sl-format-date
-                                    lang=${getLocale()}
-                                    class="truncate"
-                                    date=${org.subscription.futureCancelDate}
-                                    month="long"
-                                    day="numeric"
-                                    year="numeric"
-                                  >
-                                  </sl-format-date>`,
-                              )}
+                              ${org.subscription.status ===
+                              SubscriptionStatus.Trialing
+                                ? msg(
+                                    html`Your trial will end on
```

Comment:
I feel like this could be a separate message function instead of wrapping the entire format date?

Reply:
yeah, you're right, factored this out into separate template, think that will be better, though this is supported.
may need to do a pass on more strings like this